tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1999,6 +2000,100 @@ public:
                                                     convertVector2RVec);
    }
 
+   /////////////////////////////////////////////////////////////////////////////
+   /// \brief Return information about the dataframe.
+   /// \return information about the dataframe as string
+   ///
+   /// This convenience function describes the dataframe and combines following information:
+   /// - Number of event loops run, see GetNRuns()
+   /// - Number of total and defined columns, see GetColumnNames() and GetDefinedColumnNames()
+   /// - Name of the columns, see GetColumnNames()
+   /// - Type of the columns, see GetColumnType()
+   /// - Number of processing slots, see GetNSlots()
+   ///
+   /// This is not an action nor a transformation, just a query to the RDataFrame object.
+   ///
+   /// Please note that this is a convenience feature and the layout of the output can be subject
+   /// to change and should not be automatically parsed.
+   ///
+   /// ### Example usage:
+   /// ~~~{.cpp}
+   /// RDataFrame df(10);
+   /// auto df2 = df.Define("x", "1.f").Define("s", "\"myStr\"");
+   /// // Describe the dataframe
+   /// std::cout << df2.Desribe() << std::endl;
+   /// ~~~
+   ///
```

Comment:
should mention differences between calling `Describe` on the head node or on other nodes, following GetDefinedColumnNames or others

Reply:
You mean that it returns just the state up to the selected node? Or is there any other detail I've missed?

Reply:
See the sentence added above.

Reply:
Last comment standing!